.github/workflows/multi-node-test-workflow.yml

```diff
@@ -27,4 +27,4 @@ jobs:
       - name: Checkout Branch
         uses: actions/checkout@v2
       - name: Run integration tests with multi node config
-        run: ./gradlew integTest -PnumNodes=3 -Dopensearch.version=1.3.0-SNAPSHOT
\ No newline at end of file
+        run: ./gradlew integTest -PnumNodes=3 -Dopensearch.version=1.3.1-SNAPSHOT
```

Comment:
Hi @bowenlan-amzn @lezzago , do you plan to remove the opensearch.version property from all files and let it use default set here [build.gradle](https://github.com/opensearch-project/alerting/pull/364/files#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7R10)

```java
opensearch_version = System.getProperty("opensearch.version", "1.3.1-SNAPSHOT")
```

Reply:
Should definitely remove it, it's redundant.